packages/moxb/src/routing/url-arg/__tests__/UrlArgTypeOrderedStringArray.test.ts

```diff
@@ -18,8 +18,31 @@ describe('URL arg type boolean', () => {
         expect(v1).toBe(v2); // The objects are actually the same, not only equal!
     });
 
+    it('should not memorize values!', () => {
+        const v1 = parser('a,b,c', ['42']);
+        v1[1] = 'OOPS';
+        const v2 = parser('a,b,c', ['42']);
+        expect(v1).toEqual(['a', 'OOPS', 'c']);
+        expect(v2).toEqual(['a', 'b', 'c']);
+    });
+
+    it('should return the same array instance on subsequent calls with the same string for the same param', () => {
+        const v1 = typeDef.getParser('test')('a,b,c', ['42']);
+        const v2 = typeDef.getParser('test')('a,b,c', ['42']);
+        expect(v1).toBe(v2); // The objects are actually the same, not only equal!
+    });
+
+    it('should return the same array instance on subsequent calls with the same string for the different params', () => {
+        const v1 = typeDef.getParser('test')('a,b,c', ['42']);
+        const v2 = typeDef.getParser('x')('a,b,c', ['42']);
+        expect(v1).not.toBe(v2); // The objects are actually not the same
+        expect(v1).toEqual(v2); // ...but equal!
+    });
+
     it('should be able to parse empty strings', () => {
-        expect(parser('', ['42'])).toEqual([]);
+        expect(parser('', ['42'])).toEqual(['42']);
+        const defaultValue = ['1'];
+        expect(parser('', defaultValue)).toBe(defaultValue);
     });
```

Comment:
Again, because of how the UrlArg code is works, we never attempt to parse empty strings, since empty args are considered non-existent. So I don't think we need this.